packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -673,6 +673,26 @@ export class PrismicRepositoryManager extends BaseManager {
 		}
 	}
 
+	async setDefaultMasterLocale(): Promise<void> {
+		const repositoryName = await this.project.getRepositoryName();
+
+		const url = new URL("repository/locales", API_ENDPOINTS.LocaleService);
+		url.searchParams.set("repository", repositoryName);
+
+		const res = await this._fetch({
+			url,
+			method: "POST",
+			body: {
+				id: "en-us",
+				isMaster: true,
+			},
+		});
+
+		if (!res.ok) {
+			throw new Error("Failed to set default master locale.");
```

Comment:
💡 I think we should return also the cause:

```suggestion
		const text = await res.text();
		throw new Error("Failed to set default master locale.", { cause: text });
```

Not done for a lot of calls in the project I see, but it's better.